components/drops/useDrops.ts

```diff
@@ -377,6 +377,7 @@ export const useHolderOfCollectionDrop = () => {
 export const useRelatedActiveDrop = (collectionId: string, chain: Prefix) => {
   const { drops } = useDrops({
     chain: [chain],
+    limit: 1000,
```

Comment:
I'd rather filter in the worker side

```suggestion
    collections: [collectionId],
```

Reply:
thanks, I just realize I have already implemented the `collection` query on the worker side : ) Updated